src/main/java/com/artipie/rpm/meta/XmlMetaJoin.java

```diff
@@ -79,8 +98,34 @@ public void merge(final Path target, final Path part) throws IOException {
             throw new IOException(ex);
         }
         Files.move(res, target, StandardCopyOption.REPLACE_EXISTING);
-        Logger.info(
-            this, "%s and %s merged in %[ms]s", target.toString(),
+        Logger.debug(
+            this, XmlMetaJoin.LOG, target.toString(),
+            part.toString(), System.currentTimeMillis() - start
+        );
+    }
+
+    /**
+     * Appends data from part to target.
+     * @param target Target
+     * @param part File to append
+     * @throws IOException On error
+     */
+    @SuppressWarnings({"PMD.PrematureDeclaration", "PMD.GuardLogStatement"})
+    public void fastMerge(final Path target, final Path part) throws IOException {
```

Comment:
@olenagerasimova is there any reason (e.g. backwards compatibility) why we need _both_ `merge` and `fastMerge`? Maybe we can deprecate the former or add a puzzle to remove it?